server/planning/events.py

```diff
@@ -435,3 +441,14 @@ def generate_recurring_dates(start, frequency, interval=1, until=None, byday=Non
         return (tz.localize(dt).astimezone(pytz.UTC).replace(tzinfo=None) for dt in dates)
     else:
         return (date for date in dates)
+
+
+def setRecurringMode(event):
```

Comment:
Are you talking about the fact that the client perform a PATCH when the object contains a `_links` and the request will contain only fields with a value set ? Then up to now, in order to reset a field we give the value `null` to this field. On the client side, because this is the client's `api` service that behaves this way.